### PR TITLE
Refresh Html Css

### PR DESCRIPTION
**Pull request by at-vudang95**

### CODE REVIEW

File: public/js/main.js [hunk 1]
```diff
@@ -1,25 +1,22 @@
...
+        $('#modal-confirm-body').html(body);
+        $('#modal-confirm').modal('show');
+        $("#btn-modal-submit").one("click", function(){
+            form.submit();
+            $('#modal-confirm').modal('hide');
+        })
     });
     $("#date-sort").change(function(){
-        window.location = '/'+$("#date-sort").attr("data-table")+'/?date='+$("#date-sort").val();
+        window.location = '/'+$("#date-sort").blur("data-table")+'/?date='+$("#date-sort").val();
```
Comment (ghost) [inline public/js/main.js]: Change:
```window.location = '/'+$("#date-sort").blur("data-table")+'/?date='+$("#date-sort").val();```
To: 
```window.location = '/'+$(this).blur("data-table")+'/?date='+$(this).val();```

File: public/js/main.js [hunk 2]
```diff
@@ -1,25 +1,22 @@
...
     });
     $("#text-sort").change(function(){
         window.location = '/'+$("#text-sort").attr("data-table")+'?keyword='+$("#text-sort").val();
     });
```
Comment (ghost) [inline public/js/main.js]: Change:
```window.location = '/'+$("#text-sort").attr("data-table")+'?keyword='+$("#text-sort").val();```
To:
```window.location = '/'+$(this).attr("data-table")+'?keyword='+$(this).val();```

File: resources/views/categories/index.blade.php [hunk 3]
```diff
@@ -3,10 +3,10 @@
 @section('main-content')
     <div class="box" xmlns="http://www.w3.org/1999/html">
```
Comment (ghost) [inline resources/views/categories/index.blade.php]: Why add: 
```xmlns="http://www.w3.org/1999/html"```
in here ?

**Review by ghost — COMMENTED** [review]
Pls check my comment

File: resources/views/categories/index.blade.php (hunk 3 above)
Comment (at-vudang95) [reply resources/views/categories/index.blade.php]: I will remove it.

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**